README.md

```diff
@@ -1,57 +1,65 @@
 # Project Name
 
-> One paragraph statement about the project.
+> This project is a clone of "New York Times Magazine" completed based on the requirement to complete Microverse program/course titled "HTML and CSS".
 
 ![screenshot](./app_screenshot.png)
 
 Additional description about the project and its features.
 
+This project was designed to fulfill the following concepts:
+
+- Embedding images
+- Working with forms (buttons)
+- Floats and positioning
+- The box model
+- Grids
+- Flexbox
+
+
 ## Built With
 
-- Major languages,
-- frameworks,
-- technologies used
+- HTML
+- CSS,
 
-## Live Demo
+## Live Demo pending approval....
 
 [Live Demo Link](https://livedemo.com)
-
+[Github Repository Link](https://github.com/Rocio01/PositioningandFloatingElements/tree/feature-branch)
```

Comment:
Also, your README file doesn't contain a live demo link. Whether your project is approved or not you must put a live demo link. Update the live demo link every time you make any changes.